packages/particle-network/src/index.ts

```diff
@@ -0,0 +1,110 @@
+import {
+	WalletInit,
+	EIP1193Provider,
+	ProviderRpcError,
+	ProviderRpcErrorCode,
+	ProviderAccounts,
+} from '@web3-onboard/common';
+import { createEIP1193Provider } from '@web3-onboard/common';
+import { ParticleNetwork, Config } from '@particle-network/auth';
+import { ParticleProvider } from '@particle-network/provider';
+
+type AuthTypes =
+	| 'email'
+	| 'phone'
+	| 'google'
+	| 'apple'
+	| 'twitter'
+	| 'facebook'
+	| 'microsoft'
+	| 'linkedin'
+	| 'github'
+	| 'twitch'
+	| 'discord';
+
+interface PreferredAuthType {
+	type: AuthTypes;
+	setAsDisplay: boolean;
+}
+
+interface ParticleAuthModuleOptions extends Config {
```

Comment:
Not a requirement but adding JSDocs comments to these types allows for better devEx and something we strive for